python/cugraph/cugraph/datasets/metadata/karate.yaml

```diff
@@ -0,0 +1,24 @@
+name: karate
+file_type: .csv
+author: Zachary W.
+url: https://raw.githubusercontent.com/rapidsai/cugraph/branch-22.08/datasets/karate.csv
```

Comment:
Should the url be pointing at the 22.08 version of the data? What happens if the data is updated?

Reply:
You're right. This should point to the new 23.08 version of the data. I'm not sure if the data has changed at all but it is a misleading URL. Perhaps we should revisit hosting a folder of reliable datasets in our S3 bucket. I'll raise and issue for this and get this updated to the current branch for now.

Reply:
[Here](https://github.com/rapidsai/ops/issues/2737) is the ops issue that will upload dataset files to s3